teams/tests.py

```diff
@@ -0,0 +1,21 @@
+from .models import Teams
+import pytest
+
+
+@pytest.fixture
+def create_team():
```

Comment:
in this fixture, you create a specific team object 
so I think it will be better to call it like:
```suggestion
def team1():
```